resources/views/dinnerform/dinnerform_block.blade.php

```diff
@@ -0,0 +1,40 @@
+<div class="card mb-3 leftborder leftborder-info">
+    <a style="text-decoration: none !important;"
```

Comment:
There is a bootstrap thing to make a link in a card work for the whole card, that's probably a cleaner way to do this. https://getbootstrap.com/docs/4.4/utilities/stretched-link/